src/status_im/contexts/communities/actions/addresses_for_permissions/view.cljs

```diff
@@ -45,7 +46,7 @@
       :container-style {:margin-bottom 8}}]))
 
 (defn view
-  []
+  [{:keys [scroll-enabled on-scroll]}]
```

Comment:
nit: scroll-enabled?

I would also recommend passing the deref'd value instead here. 
It is less clear when the prop is a ratom and generally I think we want to avoid passing them along unless the value is being updated in the consuming component/view. Even then it can be better to pass a function to update it rather than the  ratom itself.

Reply:
@J-Son89 this is how bottom-sheet implemented, should i change this component? and can you give me example how to passing the deref'd value? 
https://github.com/status-im/status-mobile/blob/f762cdcf9bb66d336df65194cef637044c1360f8/src/status_im/common/bottom_sheet_screen/view.cljs#L90

Reply:
thanks @mohsen-ghafouri - why can't we just do
`:scroll-enabled @scroll-enabled` ? 🤔 

of course these changes will probably affect other places that we probably need to adjust too. 
@smohamedjavid any idea if we need to pass this ratom for a particular reason?

Reply:
I'm a bit afraid of unknown side effect as this is a basic component. but as you have more experience and if it safe i can change it to ```:scroll-enabled? @scroll-enabled?``` in [bottom_sheet_screen/view.cljs](https://github.com/status-im/status-mobile/blob/f762cdcf9bb66d336df65194cef637044c1360f8/src/status_im/common/bottom_sheet_screen/view.cljs#L90) also update all usages in other screens

Reply:
seems @briansztamfater added this value - https://github.com/status-im/status-mobile/pull/16319/files

From that pr it looks like it should have been done this way initially. I would be reasonably confident it's safe to update it.

Reply:
Great, let me update and test it again

Reply:
@J-Son89 resolved